### DISCUSSION TITLE
Set colors with vim.inspect()

### DISCUSSION THREAD

**Discussion by samEngineering** [kyazdani42/nvim-web-devicons#276] [Q&A]
Does anyone know a way to set colors using a (built-in) variable with `vim.inspect()`? I have the following in my config but it is throwing an error.
```
devicons.setup({
   color_icons = true,
   default = true,
   override = {
      py  = { icon="", color=vim.inspect(Yellow), name="Python" },
   }
})
```
Thanks :)

**Comment by alex-courtis** [comment]
You can just set `color = "Yellow",`

**Comment by samEngineering** [comment]
True, this will grab the built in `Yellow` in `nvim`. If the `Yellow` variable is reassigned though, it still produces the default yellow. As there a way to grab a variable set by the user? Thanks :)

**Reply by gegoune** [reply]
Make sure you run setup() after you set necessary variables.

**Reply by alex-courtis** [reply]
I use environment variables for my base16 colours, see https://github.com/alex-courtis/arch/blob/2a4534d55d2bf1f759a64e468724f5e9e980fee1/config/nvim/lua/amc/init/appearance.lua 

```sh
:; env | grep BASE16
BASE16_BLUE=#6fb3d2
BASE16_BROWN=#be643c
BASE16_COMMENTS=#b0b0b0
BASE16_CYAN=#76c7b7
BASE16_DARK_FOREGROUND=#d0d0d0
BASE16_DEFAULT_BACKGROUND=#000000
BASE16_DEFAULT_FOREGROUND=#e0e0e0
...
```

**Comment by samEngineering** [comment]
I do something similar, but I use `os.getenv()`. I've never seen `vim.env` before, I like that a bit better.
